web/client/actions/featuregrid.js

```diff
@@ -204,11 +204,11 @@ function setLayer(id) {
         id
     };
 }
-function updateFilter(update) {
+function updateFilter(update, dontUpdateQuery) {
     return {
         type: UPDATE_FILTER,
-        update
-
+        update,
+        dontUpdateQuery
```

Comment:
Please let me know if we can move this logic  inside the epic and avoid this flag to be passed through components